utilities/cassandra/format.cc

```diff
@@ -112,7 +116,11 @@ void ExpiringColumn::Serialize(std::string* dest) const {
   rocksdb::cassandra::Serialize<int32_t>(ttl_, dest);
 }
 
-std::unique_ptr<ExpiringColumn> ExpiringColumn::Deserialize(
+bool ExpiringColumn::Collectable() const {
+  return Timestamp() + ttl_ * 1000000 < time(NULL) * 1000000;
```

Comment:
1. not sure if we should use c++ 11 time style here: http://en.cppreference.com/w/cpp/chrono/time_point
2. I believe we should turn it into a tombstone first, then compaction filter kicks in according to gc_grace_period.